### ISSUE TITLE
Robust factor analysis

### ISSUE THREAD

**Issue by DominiqueMakowski** [easystats/parameters#225]
Now that correlation is on CRAN, we could add robust=F option to factor_analysis so that it would be based on a robust correlation method (spearman, percentage bend, sheperd's pi or biweight via `robust_method`).

**Comment by mattansb** [comment]
The idea of "robust" is weird - robust measures are measures less affected by changes in data values, which Spearman's corr is because it is on ranked data. But it is not like it gives the same info as Pearson's corr - both test different things. Just like the median isn't a "robust" mean...

All this to day - are we documenting well enough for this?

---

Also, yes, "robust" PCA/FA sounds like a great idea😜

**Comment by DominiqueMakowski** [comment]
> The idea of "robust" is weird - robust measures are measures less affected by changes in data values, which Spearman's corr is because it is on ranked data. But it is not like it gives the same info as Pearson's corr - both test different things. Just like the median isn't a "robust" mean...

Yes that's very true, although the rank conversion can eventually mitigate for the effect of outliers (let's say you have the same correlated data, spearman and Pearson giving roughly the same results, if you add one big outlier, the Pearson correlation will take off, while Spearman will remain mostly calm ☮️), so in that sense, it is sort of a *robust* correlation method? but then what does robust even mean... anyway that's eventually a discussion to have on correlation 😁

**Comment by strengejacke** [comment]
Is this still important for _parameters_, or something for correlation?
